Gameplay/EnemyController.cpp

```diff
@@ -21,13 +21,22 @@ void Hachiko::Scripting::EnemyController::OnAwake()
 {
 	game_object->GetComponent<ComponentAgent>()->AddToCrowd();
 	_attack_range = 1.5f;
+	_combat_stats = game_object->GetComponent<Stats>();
+	_combat_stats->_attack_power = 1;
+	_combat_stats->_attack_cd = 1;
+	_combat_stats->_move_speed = 4;
+	_combat_stats->_max_hp = 4;
+	_combat_stats->_current_hp = _combat_stats->_max_hp;
 	_stun_time = 0.0f;
 	_is_stunned = false;
+
 }
 
 void Hachiko::Scripting::EnemyController::OnStart()
 {
 	// TODO: Find by name in scene.
+	if (!_player)	_player = SceneManagement::FindInCurrentScene(12338322613321170553);
```

Comment:
Wow, this is correct? Can't we pass the player's Go as a serialized variable?

Reply:
I think this came from back when we were rushing VS2, but ever since a commit from May 23 we already pass __player_ as a serialized variable, this just tries to find it by UID in case the user forgets to drag and drop in the editor. It should have been removed back then but instead I don't know why I added this silly check, it's removed now.